boot/freeldr/freeldr/ntldr/registry.c

```diff
@@ -52,6 +52,16 @@ CmpAllocate(
     return FrLdrHeapAlloc(Size, Tag);
 }
 
+VOID
+CmpFreeEx(
+    _In_ PVOID Ptr,
+    _In_ ULONG Quota,
+    _In_ ULONG Tag)
+{
+    UNREFERENCED_PARAMETER(Quota);
+    FrLdrHeapFree(Ptr, Tag);
+}
+
 VOID
 NTAPI
 CmpFree(
```

Comment:
I assume `HvInitialize()` cannot be modified, can it?
https://github.com/reactos/reactos/blob/c6c6c62cb5b44468aad38b0604339a97f80b1b69/sdk/lib/cmlib/cmlib.h#L369

Reply:
It cannot.